commands/_mongoFunctions.py

```diff
@@ -28,10 +32,57 @@ def init():
     guild_list = list(Guilds.find({}))
 
     for guild in guild_list:
-        for key, value in guild.items():
-            if key == 'guild_id':
-                coll = GuildInformation["a" + str(value) + ".PendingVerificationUsers"]
-                coll.delete_many({})
+        Guild_Cache[str(guild['guild_id'])] = {}
+        Guild_Cache[str(guild['guild_id'])]['settings'] = guild
+
+        pending_verification_coll = GuildInformation["a" + str(guild['guild_id']) + ".PendingVerificationUsers"]
+        pending_verification_coll.delete_many({})
```

Comment:
Wait does the bot now support pending verification in case it goes offline now :O

Reply:
LOL nope that's been there forever it just purges the collection of pending verification users on bot start because we no longer have the code saved that we gave them

Reply:
Oh true ahaha